packages/react-reconciler/src/ReactFiberClassComponent.new.js

```diff
@@ -216,8 +217,9 @@ const classComponentUpdater = {
     }
 
     enqueueUpdate(fiber, update, lane);
-    const root = scheduleUpdateOnFiber(fiber, lane, eventTime);
+    const root = markUpdateLaneFromFiberToRoot(fiber, lane);
     if (root !== null) {
+      scheduleUpdateOnFiber(root, fiber, lane, eventTime);
```

Comment:
It would only happen when the component is unmounted. Those checks should probably move to `enqueueUpdate`, though, since that's where similar warnings are.